pkg/apis/serving/k8s_validation.go

```diff
@@ -109,6 +112,32 @@ func ValidateVolumes(ctx context.Context, vs []corev1.Volume, mountedVolumes set
 	return volumes, errs
 }
 
+func shouldAllowPersistenVolumeClaims(ctx context.Context, volume corev1.VolumeSource, features *config.Features) *apis.FieldError {
+	var errs *apis.FieldError
+	if volume.PersistentVolumeClaim != nil {
```

Comment:
could turn this in to a guard clause and exit immediately if nil, then the rest can be outdented